packages/web-components/fast-foundation/src/listbox/listbox.ts

```diff
@@ -81,9 +82,16 @@ export class Listbox extends FASTElement {
      */
     @observable
     public selectedOptions: ListboxOption[] = [];
-    protected selectedOptionsChanged(prev, next) {
+    protected selectedOptionsChanged(
+        prev: ListboxOption[] = [],
+        next: ListboxOption[]
+    ): void {
+        if (prev.length && next.length && !difference(prev, next).length) {
```

Comment:
How expensive is the `difference` function? Should this be moved inside the `isConnected` branch?

Reply:
This can be changed to use native methods. I'll update.